amundsen_application/static/js/pages/DashboardPage/ShimmeringDashboardLoader/index.tsx

```diff
@@ -6,16 +6,16 @@ import * as React from 'react';
 import './styles.scss';
 
 const ShimmeringDashboardLoader: React.FC = () => (
```

Comment:
Using the pattern of "ShimmeringXLoader" seems somewhat verbose with both a prefix and a suffix. How do you feel about adopting either "XLoader" or "XShimmer" for these types of components?

Reply:
What about just ShimmeringColumnLineageLoader?

Reply:
Sounds good!
Ideally, we should come up with a generic component that packs all this in one. At some point!